rsfbclient-core/src/row.rs

```diff
@@ -36,13 +36,25 @@ impl Row {
 }
 
 #[derive(Debug, Clone)]
-pub struct Column(pub Option<ColumnType>);
+pub struct Column {
+    pub value: Option<ColumnType>,
+    pub name: String,
+}
+
+impl Column {
+    pub fn new(name: String, value: Option<ColumnType>) -> Self {
+        Column {
+            name: name,
+            value: value,
+        }
```

Comment:
```suggestion
        Column {
            name,
            value,
        }
```

To make clippy happy